src/components/index/RequestsTable.tsx

```diff
@@ -0,0 +1,71 @@
+import { useState, useEffect } from "react";
+import { Body } from "components/table/Table.styled";
+import {
+  StyledCellBody,
+  StyledHeadRow,
+  StyledTableWrapper,
+  StyledRow,
+} from "./RequestsTable.styled";
+
+import useClient from "hooks/useOracleClient";
+import useReader from "hooks/useOracleReader";
+import useRequestsTableData from "./useRequestsTableData";
+
+const RequestsTable = () => {
+  const { state } = useClient();
+  const { descendingRequests } = useReader(state);
+  const [requeryData, setRequeryData] = useState(true);
+  const { headerCells, rows } = useRequestsTableData(
+    descendingRequests,
+    requeryData,
+    setRequeryData
+  );
+
+  // Re-renders table too quickly. Slow down renders from requests.
+  useEffect(() => {
+    const interval = setInterval(() => {
+      setRequeryData(true);
+    }, 10000);
+
+    return () => clearInterval(interval);
+  }, [setRequeryData]);
```

Comment:
i highly suggest removing this, i just dont think its necessary if we further refactor the useRequestsTableData to not have a useEffect

Reply:
Removed due to pure function.